oeps/processes/oep-0054-core-contributors.rst

```diff
@@ -189,6 +189,10 @@ The process for nomination is as follows:
 
 #. During the comment period, other CCs should weigh in on the thread. As much
    as possible, posts should conclude with a definitive "yes" or "no" vote.
+   "Liking" the nomination post does not count as a vote.
+
+   #. Anyone with commit access to the relevant repos may vote. This may change
+      as the Core Contributor program evolves.
```

Comment:
Hmm. I don't _fully_ agree that you have to have commit access to the repo. If you're a CC on other repos, but have say pulled into or reviewed the candidate's work previously, I feel you should be able to give a vote with explanation.

I absolutely _do not_ agree that you need commit rights to post a No vote, especially in cases of CoC violations or otherwise not upholding community values.

Finally, this doc is very explicitly written to be agnostic to CC roles. So I'd think about how you could rephrase. 

Combining all that feedback, maybe something like:

```
In the case of Core Contributor roles that involve granting commit access to repos, anyone who has commit access to the relevant repos may vote; additionally, all core contributors and indeed any community member are welcome to provide comments on the thread.
```

Reply:
FWIW, I'm actually open to a wide range of possibilities here, so I ask the following only for clarification (and not because I'm trying to argue against it). Is this an accurate summary of your view?

1. Any member of the community is free to make comments in the thread, to either voice support or concerns.
2. Any core contributor to Open edX may vote, regardless of whether they are formally a part of the CC program today, or are de facto contributors (e.g. edX employees with merge rights).
3. It is up to the core contributors themselves to judge whether or not they are qualified to vote, based on the nature of their interactions with the nominee. If the link is not obvious, the CC should provide some explanation with their vote.

> I absolutely do not agree that you need commit rights to post a No vote, especially in cases of CoC violations or otherwise not upholding community values.

In this case, you'd still want them to be a core contributor of some sort to have a "no" vote, right? I'm interpreting this to mean, "I think it would be good for CCs to be able to vote 'yes' on a candidate even if those CCs aren't working on those particular repos (or work on entirely different things), but I might be convinced otherwise. In contrast, I believe _very strongly_ that any CC should be able to vote 'no' based on their community interactions with this person." I ask this clarifying question because I'm not sure whether you intended to make sure the "no" vote could be open to the full range of CCs, or whether you intended to open it up to the full community at large.

Reply:
Great question, and summary overall that crystalizes my thinking. I think anyone in the community should be able to voice concerns (ex, a new dev who's not a CC who's had particularly sour interactions with the nominee). I think the sponsor should take all concerns seriously and follow up on allegations. To me, if I nominated someone and got feedback like that, I would consider it a "no" vote (although possibly pending asking for proof that this occurred). Or I might expect the sponsor to raise allegations like this to the program admins or via the [code of conduct escalation path](https://openedx.org/code-of-conduct/). I wouldn't approve the nominee until concerns were addressed sufficiently - perhaps it's a misunderstanding, perhaps it's a serious CoC violation, or somewhere in between.

Kind of a rambly paragraph. But I would hope that a non-CC raising a serious concern would be seen as a de facto No vote. I think also a No vote is a start of a conversation, and it's quite possible that concerns could be withdrawn via conversation.

Reply:
Maybe it's something like, "only CC's get No votes, but CoC violations or other behaviors not upholding the "Conduct" quality of the 3 Cs ought to be raised by anyone, whether in the Discourse thread or to the Program Admins"

How does that feel?